components/tilelayout/overview.md

```diff
@@ -10,223 +10,135 @@ position: 0
 
-You do not, however, have to set `Width` and `Height` - the main measure is the `Columns` and it will suffice to create a layout.
+## State
 
-Since the Tile Layout is a block element, its width defaults to `auto` in the browser, and the actual width is distributed evenly between the number of `Columns`. Setting `Height="100%"` can let the component take up its parent dimensions in terms of height as well.
+The [Tile Layout allows getting and setting its state]({%slug tilelayout-state%}). The TileLayout state contains information about the tiles' order, column span and row span.
 
-If the width and height dimensions are insufficient to accommodate the defined row height and column width that the current tiles create, the actual row height and/or column width will decrease so that the appointed number of columns fit in the available width and the existing number of rows fit in the available height.
 
-`Columns`, `Width` and `Height`  have no default values.
+## Events
 
-`ColumnWidth` and `RowHeight` default to `1fr`. Thus, you may want to omit setting a `ColumnWidth` in order to get a more responsive (fluid) layout - all columns will then have the same width - a fraction of the total width.
+The Tile Layout fires [events when the user resizes or rearranges tiles]({%slug tilelayout-events%}). This allows custom logic execution, refreshing of nested components and saving the [TileLayout state]({%slug tilelayout-state%}) for later restore.
 
-There are two other settings you should take into account if you set explicit dimensions to the main element - the `ColumnSpacing` and `RowSpacing` - they are CSS units that define the gaps between the individual columns and rows and count towards the total dimensions of the component. They default to `16px`.
 
-Lastly, you can also set the `Class` parameter that renders at the main wrapping element of the tile layout so you can cascade custom CSS rules through it.
+## TileLayout Parameters
 
-### Individual Tiles
+The following table lists the Tile Layout parameters. Also check the [TileLayout API Reference](/blazor-ui/api/Telerik.Blazor.Components.TelerikTileLayout) for a full list of all properties, methods and events.
 
-Each tile provides settings that define how many columns and rows its takes up - the `ColSpan` and `RowSpan` parameters. It also provides a `Class` parameter so you can cascade CSS rules through it.
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
 
-The largest `ColSpan` any tile has determines the lowest number of `Columns` the entire layout will actually have - for example, if you have a tile with `ColSpan=2` and `Columns=1`, there will still be two columns.
+| Parameter | Type and Default&nbsp;Value | Description |
+| --- | --- | --- |
+| `Class` | `string` | Renders a custom CSS class to the `<div class="k-tilelayout">` element. Use it to [override theme styles]({%slug themes-override%}). |
```

Comment:
This part should be reworded for consistency. As agreed, the descriptions should start with a noun.